js/pages/cohort-definitions/components/checks/conceptset-warnings.js

```diff
@@ -91,19 +91,23 @@ define(['knockout', 'text!./conceptset-warnings.html',
       };
 
       self.getWarnings = function() {
-        if (parseInt(self.cohortDefinitionId(), 10) <= 0 || isNaN(self.cohortDefinitionId())) {
-          return false;
+        if (this.cohortDefinitionId()) {
```

Comment:
cohortDefinitionId becomes undefined after you try to save newly created concept so the check will not fire any more after changing the cohort definition

Reply:
fixed